build.js

```diff
@@ -143,7 +142,7 @@ function buildlocale (source, locale) {
     .use(layouts({
       engine: 'handlebars',
       pattern: '**/*.html',
-      partials: mapHandlebarsPartials(metalsmith, 'layouts', 'partials'),
+      partials: 'layouts/partials',
```

Comment:
`metalsmith-layouts` handles the paths. See https://github.com/superwolff/metalsmith-layouts/blob/master/lib/helpers/read-partials.js#L22

`partials: path.join(__dirname, 'layouts', 'partials'),`
Does **not** work.